earthfile2llb/converter.go

```diff
@@ -496,7 +497,7 @@ func (c *Converter) RunExitCode(ctx context.Context, commandName string, args, m
 		"%s %s%s%s",
 		commandName, // eg IF
 		strIf(privileged, "--privileged "),
-		strIf(noCache, "--no-cache "),
+		strIf(noCache, "--symlink-no-cache "),
```

Comment:
Probably unintentional? :-)

Reply:
ouch, that was a bad find-and-replace. Good catch.